runtime/src/config.c

```diff
@@ -212,8 +213,18 @@ void printConfigVarTable(void) {
         }
         fprintf(stdout, "  %*s: ", longestName, configVar->varName);
         fprintf(stdout, "%s", configVar->defaultValue);
-        if (configVar->setValue) {
-          fprintf(stdout, " (configured to %s)", configVar->setValue);
+        if (configVar->setValue || configVar->private) {
+          fprintf(stdout, " (");
+          if (configVar->private) {
+            fprintf(stdout, "private");
+            if (configVar->setValue) {
+              fprintf(stdout, ", ");
+            }
+          }
+          if (configVar->setValue) {
+            fprintf(stdout, "configured to %s", configVar->setValue);
```

Comment:
I expected to see a test updated/added in response to this change, but I don't see one.

Reply:
Assuming you're talking about the labeling of configs as private in the help output, the last case (`--help` / `privateconfigs-help.good`) in `test/execflags/configs/privateconfigs.chpl` is meant to cover this case.

Reply:
I was expecting to see something with "(private, configured to " in it, based on how this code looks, but I don't see anything like that

Reply:
Oh that.  That's old code that didn't really "change" so much as moved (line 216 prior to this effort became line 225).  The reason for the refactoring was just to use the same parens for either message being printed out.  There are a bunch of tests in test/execflags/shannon/configs/help/ that ensure it's still working.

Reply:
Ah, but now I understand that you're saying you'd like to see a test that shows both messages.  That seems appropriate, let me add one.